icons/save-all.svg

```diff
@@ -0,0 +1,16 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M20.2 18H7.8c-1 0-1.8-.8-1.8-1.8V3.8C6 2.8 6.8 2 7.8 2H18l4 4v10.2c0 1-.8 1.8-1.8 1.8z" />
+  <polyline points="18 18 18 11 10 11 10 18" />
+  <polyline points="10 2 10 6 16 6" />
+  <path d="M18 22H4c-1.1 0-2-.9-2-2V6" />
```

Comment:
```suggestion
  <path d="M20.2 18a1.8 1.8 0 0 0 1.8-1.8V6l-4-4H7.8C6.8 2 6 2.8 6 3.8v12.4c0 1 .8 1.8 1.8 1.8Z" />
  <path d="M10 2v4h6" />
  <path d="M18 18v-7h-8v7" />
  <path d="M18 22H4a2 2 0 0 1-2-2V6" />
```

Reply:
To piggyback off of this one, I feel like a 2px radius fits better than the current 1.8.
```suggestion
  <path d="M6 4a2 2 0 0 1 2-2h10l4 4v10.2a2 2 0 0 1-2 1.8H8a2 2 0 0 1-2-2Z" />
  <path d="M10 2v4h6" />
  <path d="M18 18v-7h-8v7" />
  <path d="M18 22H4a2 2 0 0 1-2-2V6" />
```